### PR TITLE
cosuming of AccountUUID works on webapp

### PR DESCRIPTION
**Pull request by leonPep42**

### CODE REVIEW

File: client-side/src/app/card/card.component.ts [hunk 1]
```diff
@@ -44,11 +45,21 @@ export class CardComponent implements OnInit {
...
+        const varData = card.variablesData[varName];
+        if(varData.source == 'Variable') {
+            values[varName] = (this.parameters && this.parameters[varData.value]) ? this.parameters[varData.value] : '0';
+        } else {
+            values[varName] = varData.value;
+        }
+    }
+    for (const varName in card.benchmarkVariablesData) {
+        const varData = card.benchmarkVariablesData[varName];
+        if(varData.source == 'Variable') {
```
Comment (ShirMarciano) [inline client-side/src/app/card/card.component.ts:57]: this code (57-61) is the same code as in line 48 and as in the benchmarkchart file (only parameters name different ) 
If you will have a bug or a change you will need to do - you will need to remember to do it in al places

File: client-side/src/app/chart/chart.component.ts [hunk 2]
```diff
@@ -51,7 +54,12 @@ export class ChartComponent implements OnInit {
     // sending variable names and values as body
     let values = {};
     for (const varName in configuration.variablesData) {
-      values[varName] = (configuration.variablesData[varName].source == 'Variable') ? configuration.variablesData[varName].valueFromPage : configuration.variablesData[varName].value;
+      const varData = configuration.variablesData[varName];
+      if(varData.source == 'Variable') {
```
Comment (ShirMarciano) [inline client-side/src/app/chart/chart.component.ts:58]: Same code as above also - need to reduce the code duplication

File: client-side/src/app/scorecards-editor/scorecards-editor.component.ts [hunk 3]
```diff
@@ -208,21 +201,13 @@ export class ScorecardsEditorComponent implements OnInit {
...
-            this._pageConfiguration.Parameters.push({
-                Key: param,
-                Type: 'String',
-                Consume: true,
-                Produce: false
-            });
-        }
-
+        this._pageConfiguration.Parameters.push({
+            Key: 'AccountUUID',
```
Comment (ShirMarciano) [inline client-side/src/app/scorecards-editor/scorecards-editor.component.ts:206]: this is the hardcoded parameter you put in the meanwhile?  -we need a comment here for explanation
what was the code you removed?

File: client-side/src/app/table/table.component.ts [hunk 4]
```diff
@@ -91,7 +94,12 @@ export class TableComponent implements OnInit {
     // sending variable names and values as body
     let values = {}
     for(const varName in configuration.variablesData) {
-        values[varName] = configuration.variablesData[varName].value
+      const varData = configuration.variablesData[varName];
```
Comment (ShirMarciano) [inline client-side/src/app/table/table.component.ts:97]: again - same code
You worked so Hard to remove the code duplication we had before, please continue to keep on that

File: client-side/src/app/scorecards-editor/scorecards-editor.component.ts (hunk 3 above)
Comment (leonPep42) [reply client-side/src/app/scorecards-editor/scorecards-editor.component.ts:206]: the code removed was something irrelevant I copied from Avner related to scripts and forgot to delete

File: client-side/src/app/chart/chart.component.ts [hunk 5]
```diff
@@ -51,7 +54,12 @@ export class ChartComponent implements OnInit {
     // sending variable names and values as body
     let values = {};
     for (const varName in configuration.variablesData) {
-      values[varName] = (configuration.variablesData[varName].source == 'Variable') ? configuration.variablesData[varName].valueFromPage : configuration.variablesData[varName].value;
+      const varData = configuration.variablesData[varName];
```
Comment (ShirMarciano) [inline client-side/src/app/chart/chart.component.ts:57]: You can use here also the "buildVariableValues"

Comment (leonPep42) [reply client-side/src/app/chart/chart.component.ts:57]: I'm currently working on another glitch in this file, will push it soon

**Pull request merged**